libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2247,6 +2249,15 @@ MAV_RESULT GCS_MAVLINK::handle_command_set_message_interval(const mavlink_comman
     return MAV_RESULT_FAILED;
 }
 
+MAV_RESULT GCS::set_message_interval(uint8_t channel, uint32_t msg_id, int32_t interval_us)
+{
+    if (channel < MAVLINK_COMM_NUM_BUFFERS) {
+        return _chan[channel]->set_message_interval(msg_id, interval_us);
```

Comment:
@WickedShell pointed out that this probably needs a nullptr check.

Reply:
I fixed it